index.html

```diff
@@ -3,40 +3,98 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+<nav id = "navbar">
+  <img src="/images/spotify-logo.png" alt="spotify logo">
+  <ul>
+    <li>
+      <a href="#">Premium</a>
+    </li>
+    <li>
+        <a href="#">Discover</a>
+    </li>
+    <li>
+        <a href="#">Help</a>
+    </li>
+    <li>
+        <a href="#">Download</a>
+    </li>
+  </ul>
+</nav>  
+
+<section id="landing">
+  <h1>Music for everyone.</h1>
+  <p>Spotify is now free on mobile, tablet and computer. <br>
+  Listen to the right music, wherever you are.</p>
+</section> 
+
+<section id="middle">
```

Comment:
for the name of the id, try to give it more meaning than the position it belongs now, like 'info-container' for the simple reason that if in the future you change the code and add an other article, this one won't be in the middle anymore and it would be confusing or you would need to review your code